### ISSUE TITLE
AttributeError: Can't get attribute 'Hardswish'

### ISSUE THREAD

**Issue by mowkee** [mikel-brostrom/Yolov5_DeepSort_Pytorch#41]
I am using Anaconda environment with python 3.8 and I'm getting this error.

**Comment by mikel-brostrom** [comment]
Check that you fulfill all the requirements. I seems to me that you **pytorch** is not **>=1.7**

**Comment by mowkee** [comment]
I installed the latest version even tried the nightly build of `pytorch`. I tried the same in Linux and it worked though but in Windows I have yet to make it work. Are you running this on Windows?

**Comment by mikel-brostrom** [comment]
This repo has never been run on a windows. But this issue may help https://github.com/ultralytics/yolov5/issues/773

**Comment by mowkee** [comment]
I've seen that and that's not a real solution. Telling me to use Google Colab or Kaggle Notebook or Docker because it's "Tested" is an excuse worst than the crime itself. Anyways it looks like it's a **pytorch** problem since the same code works in Linux. I've heard somewhere that a workaround would be to replace the `Hardswish` with `LeakyReLU` and retrain the network but that's a huge undertaking. Anyways thanks for the response.

**Comment by mikel-brostrom** [comment]
Yolov5 uses CI for testing its basic functionality: "including training (train.py), testing (test.py), inference (detect.py) and export (export.py) on MacOS, **Windows**, and Ubuntu"

Using a verified environment like a DockerContainer to get stuff working is a software development standard. Specially if it was already tested and recommended by its creator. I would say it is even more important if you have problems getting the code running.

**Comment by mowkee** [comment]
I resolved the issue and ran the script successfully in Windows. Initially I was trying to run the script in **PyCharm**. But this time I created a environment in **Anaconda Powershell Prompt** using: `conda create -n python*3.7 deepsort`. Then open the requirements and remove the lines `torch>=1.6.0` and  `torchvision>=0.7.0` then go to [pytorch](https://pytorch.org/get-started/locally/) and select Build, OS, Package, Language etc. Then copy the line after `Run this Command:` and paste it into your command line. At the time of writing this comment I got this line `pip install torch===1.7.0 torchvision===0.8.1 torchaudio===0.7.0 -f https://download.pytorch.org/whl/torch_stable.html`. Then follow the instructions in the repository for installing the rest of the requirements and obtaining the weights and checkpoint files and then run the `track.py` script.
